fdbserver/RocksDBCheckpointUtils.actor.cpp

```diff
@@ -143,13 +145,122 @@ Error statusToError(const rocksdb::Status& s) {
 	}
 }
 
-// RocksDBCheckpointReader reads a RocksDB checkpoint, and returns the key-value pairs via nextKeyValues.
-class RocksDBCheckpointReader : public ICheckpointReader {
+// Fetch a single sst file from storage server. The progress is checkpointed via cFun.
```

Comment:
It is a placeholder for a callback, to persist the progress of fetccKeys(), so that we don't need to start over on failures. Currently it has not been implemented.